src/pretix/base/plugins.py

```diff
@@ -56,8 +64,29 @@ def get_all_plugins(event=None) -> List[type]:
             if app.name in settings.PRETIX_PLUGINS_EXCLUDE:
                 continue
 
-            if hasattr(app, 'is_available') and event:
-                if not app.is_available(event):
+            level = getattr(app, "level", PLUGIN_LEVEL_EVENT)
+            if level == PLUGIN_LEVEL_EVENT:
+                if event and hasattr(app, 'is_available'):
+                    if not app.is_available(event):
+                        continue
+                elif organizer and hasattr(app, 'is_available'):
+                    # TODO: Re-think this behaviour. I don't think we have a better one available that's backwards-compatible?
+                    if event_fallback_used:
+                        # No events exist, no need to run query again
+                        continue
+                    event_fallback = organizer.events.first()
+                    event_fallback_used = True
+                    if not event_fallback or not app.is_available(event_fallback):
+                        continue
```

Comment:
Hm, that is not the way I suggested it. You still need to check `event_fallback` for hasattr, otherwise you only check the first plugin, all others won’t be available. So either do it as I suggested with `if not event_fallback_used:` and set the event_fallback in that if-condition. Or check above with (IMHO not as readable/understandable as my original suggestion):
```
                if (event or event_fallback) and hasattr(app, 'is_available'):
                    if not app.is_available(event or event_fallback):
```